### PR TITLE
Molecule v2

### PR DESCRIPTION
**Pull request by sbesson**
Following #16, this migrates the role to using Ansible 2.6.3/Molecule 2 as done in https://github.com/openmicroscopy/ansible-role-basedeps/pull/5#issuecomment-422020107

### CODE REVIEW

**Comment by manics** [comment]
`Nov 21 16:18:55 localhost haproxy[1714]: 127.0.0.1:52220 [21/Nov/2018:16:18:55.560] hafrontend-http habackend/<NOSRV> 0/-1/-1/-1/0 503 213 - - SC-- 1/1/0/0/0 0/0 "GET / HTTP/1.1"`
From the [haproxy docs](https://cbonte.github.io/haproxy-dconv/1.7/configuration.html#8.2.3) the diff occurs in `actconn '/' feconn` which are related to connection counts, so maybe it's a race issue? If so I think accepting `0/0` or `1/1` is an acceptable workaround.

**Comment by sbesson** [comment]
I tested Ubuntu 16.04 in isolation ([build](https://travis-ci.org/sbesson/ansible-role-haproxy/builds/458047226)) but this gave the same outcome. 3e109e8 relaxes the log line check to ignore `actonn/feconn` but I can certainly moodify to a regexp to make sure the counts are at most `1`.

**Comment by manics** [comment]
Can you make the check as tight as possible for now, and check for 0/1?

**Comment by sbesson** [comment]
See 06934b5

File: molecule/default/tests/test_default.py [hunk 1]
```diff
@@ -28,24 +29,17 @@ def test_logfiles(host, name):
...
-def test_https(host):
-    out = host.check_output("curl -ks https://localhost:443/")
-    assert '503 Service Unavailable' in out
-    assert 'No server is available to handle this request.' in out
-    with host.sudo():
-        log = host.file('/var/log/haproxy/haproxy-http.log')
-        assert log.content_string.splitlines()[-1].endswith(
-            '/0/0/0 0/0 "GET / HTTP/1.1"')
+        lastline = log.content_string.splitlines()[-1]
+        assert re.match(r'.*[0/1]/[0/1]/0/0/0 0/0 "GET / HTTP/1.1"', lastline)
```
Comment (manics) [inline molecule/default/tests/test_default.py]: No `/` in `'.*[0/1]/[0/1]`: `'.* [01]/[01]`

**Pull request merged**